frontend/app/helpers/spree/navigation_helper.rb

```diff
@@ -2,17 +2,9 @@
 
 module Spree
   module NavigationHelper
-    def spree_navigation_data
```

Comment:
@MatthewKennedy can we not remove this method as well as the initializer? this will ease the migration as developers will still be able to use this method with old templates and migrate to new nav cms gradually. We definitely should deprecate this method and remove it in Spree 5.0